src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 0.3s;
+
+    &:hover {
+      color: #{$hover-color-header};
```

Comment:
don't need tipe #{} when connect variables in one file through import, use only $ (everywhere)